ontographer/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:18 as node
```

Comment:
Please, use node:16-alpine so that we use the same Node version for TermIt and OG